### DISCUSSION TITLE
Welcome to pianfan.github.io Discussions!

### DISCUSSION THREAD

**Discussion by pianfan** [pianfan/pianfan.github.io#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cnzxb** [comment]
为什么 复刻你的 pianfan.github.io   是不是还需要配置什么才能访问  我的github.IO打开是404 呢  这个 GitHub Actions 需要配置吗

**Comment by GinsPak** [comment]
您好，请问如果想编辑标签名称的话，在代码的哪里？
